skins/oasis/modules/UserPagesHeaderController.class.php

```diff
@@ -298,102 +298,4 @@ public static function getUserURL() {
 		$userURL = $userURL[count( $userURL ) -1];
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "$userURL". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)